res/controllers/common-hid-utils.js

```diff
@@ -0,0 +1,83 @@
+HIDController.prototype.defaultScalers = {
```

Comment:
Please read my first comment again and then let me know your opinion

Reply:
The only function in this new file that I think is worth keeping is the `clearLights` function and I think it would be odd to have a file with just one function separate from common-hid-packet-parser.js.